src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
provide here modifiers 
```suggestion
    <div class="calendar calendar--start-day--mon....">
```